lib/span-data.js

```diff
@@ -82,6 +92,42 @@ SpanData.prototype.close = function() {
 };
 
 /**
+ * Trace API expects stack frames to be a JSON string with the following
+ * structure:
+ * STACK_TRACE := { "stack_frame" : [ FRAMES ] }
+ * FRAMES := { "class_name" : CLASS_NAME, "file_name" : FILE_NAME,
+ *             "line_number" : LINE_NUMBER, "method_name" : METHOD_NAME }*
+ *
+ * While the API doesn't expect a columnNumber at this point, it does accept,
+ * and ignore it.
+ *
+ * @param {string|undefined} className
+ * @param {string|undefined} methodName
+ * @param {string|undefined} fileName
+ * @param {number|undefined} lineNumber
+ * @param {number|undefined} columnNumber
+ * @constructor @private
+ */
+function StackFrame(className, methodName, fileName, lineNumber, columnNumber) {
+  if (className) {
+    this.class_name = className;
```

Comment:
Note to future me: This path will be dead code as we currently do not provide class names. It is included to be faithful to the structure expected by the server.